Source/WebKit/UIProcess/API/Cocoa/_WKTargetedElementRequest.mm

```diff
@@ -70,6 +70,25 @@ - (instancetype)initWithPoint:(CGPoint)point
     return self;
 }
 
+- (instancetype)initWithSelectors:(NSArray<NSSet<NSString *> *> *)nsSelectorsForElement
+{
+    if (!(self = [self init]))
+        return nil;
+
+    WebCore::TargetedElementSelectors selectorsForElement;
+    selectorsForElement.reserveInitialCapacity(nsSelectorsForElement.count);
+    for (NSSet<NSString *> *nsSelectors in nsSelectorsForElement) {
+        HashSet<String> selectors;
+        selectors.reserveInitialCapacity(nsSelectors.count);
+        for (NSString *selector in nsSelectors)
+            selectors.add(selector);
```

Comment:
(just thinking aloud, not needed for the patch: a `createNSSet` helper would be nice)

Reply:
Agreed! Filed: https://bugs.webkit.org/show_bug.cgi?id=274562